geometric_features/features_and_tags.json

```diff
@@ -932,6 +932,11 @@
         "Arctic",
         "Atlantic_Basin"
       ],
+      "Baffin Bay NSIDC": [
+        "Baffin_Bay_NSIDC",
+        "Arctic_NSIDC",
+        "Arctic_Basin"
```

Comment:
@milenaveneziani, I believe the main way we use the "Arctic_Basin" tag is to pull out the features that need to be `combine`d to produce the single feature that is the `Arctic_Basin`.  Would it make sense to only tag the "oceanographic" features for this purpose?  This is how the other ocean basins work.